projects/control-service/CHANGELOG.md

```diff
@@ -1,7 +1,17 @@
 Changelog
 =========
 
-1.2.14 - Next
+1.2.15 - Next
+----
+* **New feature**
+
+* **Improvement**
+
+* **Bug Fixes**
+
+* **Breaking Changes**
+
+1.2.14 - 04.10.2021
```

Comment:
I think we've missed to add  PR #325 in ChangeLog as an improvement now during install of helm chart users can specific additional labels for the control service deployment.

Reply:
I will add this.